velox/docs/configs.rst

```diff
@@ -190,3 +190,15 @@ the update mode field of the table writer operator output. ``OVERWRITE``
 sets the update mode to indicate overwriting a partition if exists.
 ``ERROR`` sets the update mode to indicate error throwing if writing
 to an existing partition.
+
+Spark-specific configuration
+----------------------------
+
+``spark.legacy-size-of-null``
```

Comment:
This name is nicer. Let's extract this change into a separate PR and also change the name in code.

Reply:
> This name is nicer. Let's extract this change into a separate PR and also change the name in code.

Hi, @mbasmanova  I extract this change into a separate PR. https://github.com/facebookincubator/velox/pull/3862